WooCommerce/Classes/ViewRelated/Dashboard/JetpackInstall/JetpackInstallSteps.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+enum JetpackInstallStep: Int, CaseIterable {
```

Comment:
nit: it would be great to add a description on top indicating where we use this enum, and what their cases represents.